products/intelprocessors.md

```diff
@@ -0,0 +1,171 @@
+    discontinued: 2018-11-01
+    eol: 2021-06-30
+    releaseDate: 2014-10-27
+
+-   releaseCycle: "Broadwell (Xeon E7v4/E7v4)"
+    discontinued: 2018-11-01
+    eol: 2022-06-30
+    releaseDate: 2014-10-27
+
+-   releaseCycle: "Broadwell DE"
+    discontinued: 2018-11-01
+    eol: 2022-12-31
+    releaseDate: 2014-10-27
+
+-   releaseCycle: "Skylake"
+    discontinued: 2019-03-04
+    eol: 2022-09-30
+    releaseDate: 2015-08-05
+
+-   releaseCycle: "Apollo Lake"
+    discontinued: false # TODO: This is likely incorrect, but I am assuming it is under production due to this CPU being launched semi recently and having its status say not discontinued: https://ark.intel.com/content/www/us/en/ark/products/195253/intel-pentium-processor-n4200e-2m-cache-up-to-2-50-ghz.html
+    eol: 2023-06-30
+    releaseDate: 2016-08-30
+
+-   releaseCycle: "Bay Trail"
+    discontinued: true
+    eol: 2021-06-30
+    releaseDate: 2013-09-11
+
+-   releaseCycle: "Braswell"
+    discontinued: true
+    eol: 2022-06-30
+    releaseDate: 2015-03-01
+
+-   releaseCycle: "Cherry Trail"
+    discontinued: true
+    eol: 2022-06-30
+    releaseDate: 2013-03-02
+
+-   releaseCycle: "Cannon Lake"
+    discontinued: 2020-02-28
+    eol: 2021-09-30
+    releaseDate: 2018-05-01
+
+-   releaseCycle: "Avoton"
+    discontinued: true
+    eol: 2021-06-30
+    releaseDate: 2013-07-01 # Not accurate, date of start of quarter 3 of 2013
+
+-   releaseCycle: "Briarwood"
+    discontinued: true
+    eol: 2021-06-30
+    releaseDate: 2013-04-01 # Not accurate, date of start of quarter 3 of 2013
+
+-   releaseCycle: "Centerton"
+    discontinued: true
+    eol: 2021-06-30
+    releaseDate: 2012-12-11
+
+-   releaseCycle: "Kaby Lake"
+    discontinued: 2020-10-09
+    eol: false
+    releaseDate: 2015-08-30
+
+-   releaseCycle: "Gemini Lake"
+    discontinued: true
+    eol: false
+    releaseDate: 2017-12-11
+
+-   releaseCycle: "Coffee Lake"
+    discontinued: 2021-12-24
+    eol: false
+    releaseDate: 2017-10-05
+
+-   releaseCycle: "Comet Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2019-08-21
+
+-   releaseCycle: "Ice Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2019-09-01
+
+-   releaseCycle: "Elkhart Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2020-09-23
+
+-   releaseCycle: "Lakefield"
+    discontinued: true
+    eol: false
+    releaseDate: 2020-06-19
+    
+-   releaseCycle: "Tiger Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2020-09-02
+
+-   releaseCycle: "Jasper Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-01-11
+
+-   releaseCycle: "Rocket Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-03-30
+
+-   releaseCycle: "Alder Lake"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-11-04
+
+---
+
+> Intel is a corporation that is known for being the world's largest semiconductor chip manufacturer. 
+
```

Comment:
Are we covering Intel Mobile CPUs completely here? For eg, I see some [CPUs here](https://compatibleproducts.intel.com/ProductDetails?activeModule=Processor%20OS%20Compatibility&prdName=Intel%C2%AE%20Core%20i7) that don't list Linux or Chrome OS - should this be covered here, or come under separate page for intel-mobile CPUs?

Reply:
Speculation but I believe that when Linux OS is not listed it's when the CPU was only sold to OEMs and none of those OEMs advertise Linux support. For example the 8565U which doesn't have Linux listed on that page does seem to work fine with the stock Linux kernel: https://linux-hardware.org/index.php?id=cpu:intel-6-142-12-core-i7-8565u